src/python/pants/backend/jvm/tasks/jar_publish.py

```diff
@@ -324,8 +324,6 @@ def register_options(cls, register):
                   'where many artifacts must align.')
     register('--transitive', default=True, type=bool,
              help='Publish the specified targets and all their internal dependencies transitively.')
-    register('--force', type=bool,
```

Comment:
This is re-instated now that the global `--force` is gone, but also deprecated with TODO to clean up custom `--force` code.